src/devices/machine/watchdog.cpp

```diff
@@ -144,14 +144,14 @@ void watchdog_timer_device::watchdog_enable(int state)
 
 void watchdog_timer_device::watchdog_fired()
 {
-	logerror("Reset caused by the watchdog!!!\n");
+	logerror("watchdog_reset issued\n");
 
 	bool verbose = machine().options().verbose();
 #ifdef MAME_DEBUG
 	verbose = true;
 #endif
 	if (verbose)
-		popmessage("Reset caused by the watchdog!!!\n");
+		popmessage("watchdog_reset issued");
```

Comment:
PR because I'm legit not sure why or how even we need this `popmessage` printed out if verbose is on. 
`popmessage` is for exceptional cases, and by quick glance a watchdog reset isn't (most NeoGeo games, airbustr ...)